### ISSUE TITLE
Synfig crashes/doesn't work as expected while animating with exported canvas

### ISSUE THREAD

**Issue by yoSachinkr** [synfig/synfig#2447]
**Describe the bug**
Synfig crashes/ doesn't work as expected while animating with exported canvas. 

**To Reproduce**
In dev version,

https://user-images.githubusercontent.com/50204208/143776059-7112f05e-3b6a-4323-9a65-f8d6943e7863.mp4

**Expected behavior**
In stable version, 

https://user-images.githubusercontent.com/50204208/143776147-4772931a-0c01-4df3-91e5-7302c491ff71.mp4

**System information:**
 - OS: Windows 10 64bit 21h2
 
 - Commit ID: SynfigStudio-1.5.1-2021.10.21-win64-2cb6c

**Comment by ice0** [comment]
@yoSachinkr Thanks for reporting, I will take a look.

**Comment by ice0** [comment]
Hi, @yoSachinkr!

Unfortunately, I was unable to reproduce the issue. 

Can you try to reproduce it again? Can it be reproduced if you save the file before the crash?
I think maybe I am missing some step here.

**Comment by yoSachinkr** [comment]
@ice0 Yes it is reproducible even after saving the file. but this time it took a while and I was able to make 3-4 keyframes in the exported canvas before the crash.

**Comment by ice0** [comment]
@yoSachinkr Can you share this saved file and show me what to do?

**Comment by yoSachinkr** [comment]
Steps:
1. Open the file.
2. Open the exported canvas.
3. Enable "animate mode" on both canvases.
4. Go to the main canvas.
5. Go to a different frame in the timeline and change the shape of the exported object.
6. Go to the exported canvas, and "add a keyframe"
7. Go back to the main canvas, change the frame in the timeline, and repeat the process.
8. After 3-4 times, synfig will crash.
[qe.zip](https://github.com/synfig/synfig/files/7692225/qe.zip)

**Comment by rodolforg** [comment]
Maybe it is related to new timetrack and #2294 fixed it

**Comment by rodolforg** [comment]
@yoSachinkr is it still reproducible in current master?

**Comment by yoSachinkr** [comment]
> @yoSachinkr is it still reproducible in current master?

@rodolforg Not sure as I'm unable to test the master.

**Comment by rodolforg** [comment]
Please try this build https://cloud.morevnaproject.org/s/dLrzniH3XQKFpkw .

**Comment by rodolforg** [comment]
@yoSachinkr please try this build: https://github.com/synfig/synfig/issues/2721#issuecomment-1172511874

**Comment by yoSachinkr** [comment]
@rodolforg I just checked, It still crashes.

**Comment by rodolforg** [comment]
Ok, this AppImage crashes here too (I realized I didn't test it before, sorry).
However, my current master build doesn't crash.
Maybe it is a package problem? @ice0 @morevnaproject 
Is it caused by the same issue discovered here https://github.com/synfig/synfig/issues/2643#issuecomment-1172865519?

**Comment by yoSachinkr** [comment]
> @yoSachinkr please try this build: [#2721 (comment)](https://github.com/synfig/synfig/issues/2721#issuecomment-1172511874)

Also this build is more prone to random crashing. 😐

**Comment by yoSachinkr** [comment]
@morevnaproject Just checked. Synfig is not crashing anymore while working with exported canvas. 😃

**Comment by rodolforg** [comment]
@morevnaproject do you know what did you change here?
